src/kakarot/accounts/account_contract.cairo

```diff
@@ -118,6 +122,40 @@ func __execute__{
     response_len: felt, response: felt*
 ) {
     alloc_locals;
+
+    // Upgrade flow
+    let (latest_account_class, latest_cairo1helpers_class) = AccountContract.get_latest_classes();
+    let (this_cairo1helpers_class) = Account_cairo1_helpers_class_hash.read();
```

Comment:
`this_cairo1_helpers_class` to be consistent with the storage var name. I4d rather call this in any cases only `helpers_class`